mws/resource_customer_managed_key.go

```diff
@@ -19,11 +19,12 @@ type AwsKeyInfo struct {
 
 // CustomerManagedKey contains key information and metadata for BYOK for E2
 type CustomerManagedKey struct {
-	CustomerManagedKeyID string      `json:"customer_managed_key_id,omitempty" tf:"computed"`
-	AwsKeyInfo           *AwsKeyInfo `json:"aws_key_info" tf:"force_new"`
-	AccountID            string      `json:"account_id" tf:"force_new"`
-	CreationTime         int64       `json:"creation_time,omitempty" tf:"computed"`
-	UseCases             []string    `json:"use_cases"`
+	CustomerManagedKeyID      string      `json:"customer_managed_key_id,omitempty" tf:"computed"`
+	AwsKeyInfo                *AwsKeyInfo `json:"aws_key_info" tf:"force_new"`
+	AccountID                 string      `json:"account_id" tf:"force_new"`
+	CreationTime              int64       `json:"creation_time,omitempty" tf:"computed"`
+	UseCases                  []string    `json:"use_cases"`
+	ReuseKeyForClusterVolumes bool        `json:"reuse_key_for_cluster_volumes,omitempty" tf:"default:true"`
```

Comment:
please bump the schema version to 2 and do schema version upgrader:

https://github.com/databrickslabs/terraform-provider-databricks/blob/bb43dd35d51870538e6670982d31f73ffb7d6e17/mws/resource_customer_managed_key.go#L103-L157

Reply:
Version 1 or 2 as current one looks to be 0? Are there docs on what the schema version upgrader is and how I can do that please?

Reply:
Ah was looking at the version within which is at 0. Will update schema version to 2

Reply:
Hi, still wanting to check how I do a schema version upgrader?

Reply:
@johnc1996 Please see the link in the first comment =)

Reply:
Have pushed a fix now and have passing tests